run-bahmni.sh

```diff
@@ -94,32 +92,30 @@ function showOpenMRSlogs {
 
 function startMart {
     echo "Starting services with profile 'bahmni-mart'..."
-    docker compose --env-file "$1" --profile bahmni-mart up -d
+    docker compose --env-file "$file" --profile bahmni-mart up -d
 }
 
 function pullLatestImages {
-    if [ "$1" == ".env" ]; then
-        echo "Pulling all 1.0.0 images..."
-    elif [ "$1" == ".env.dev" ]; then
+    if [ "$file" == ".env.dev" ]; then
         echo "Pulling all latest images..."
+        docker compose --env-file "$file" pull
     else
-        echo "[ERROR] Incorrect Environment Variable file path. Please provide either .env or .env.dev file as argument."
-        exit 1
+        echo "Pulling all 1.0.0 images..."
```

Comment:
We don't know this for sure if it is 1.0.0 or not. Just say: "Pulling all images ..."